src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -946,13 +946,13 @@
     <value>{0} uses a weak cryptographic algorithm {1}</value>
   </data>
   <data name="DoNotUseWeakKDFAlgorithm" xml:space="preserve">
-    <value>Do Not Use Weak Key Derivation Function Algorithm</value>
+    <value>Ensure Key Derivation Function algorithm is sufficiently strong</value>
```

Comment:
Can you file a docs issue for updating https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca5379? 

- https://github.com/dotnet/docs/blob/master/docs/fundamentals/code-analysis/quality-rules/ca5379.md
- https://github.com/dotnet/docs/issues

Reply:
I'll leave #4110 open until this change ships and I update the docs.